Symfony/CS/Tokenizer/Token.php

```diff
@@ -382,8 +382,8 @@ public function isStructureAlternativeEnd()
     /**
      * Check if token is a whitespace.
      *
-     * @param array  $opts                array of extra options
-     * @param string $opts['whitespaces'] string determining whitespaces chars, default is " \t\n\r\0\x0B"
+     * @param array $opts array of extra options,
```

Comment:
you need to keep as much of the next line on this line as possible, ie, only wrap at the last possible moment at 80 cahrs